datafusion/common/src/error.rs

```diff
@@ -34,7 +34,7 @@ use parquet::errors::ParquetError;
 use sqlparser::parser::ParserError;
 
 /// Result type for operations that could result in an [DataFusionError]
-pub type Result<T> = result::Result<T, DataFusionError>;
+pub type Result<T, E = DataFusionError> = result::Result<T, E>;
```

Comment:
This is the actual change as I undertand

Reply:
Cool